runner/runner/util.py

```diff
@@ -57,6 +57,18 @@ def unflatten(it: Iterable[V]) -> Iterable[Iterable[V]]:
         yield (elem,)
 
 
+def flatten_maps_list(maps: List[Mapping[str, str]], key_prefix: Optional[str] = "") -> Mapping[str, str]:
```

Comment:
```suggestion
def flatten_maps_list(maps: List[Mapping[str, V]], key_prefix: str = "") -> Mapping[str, V]:
```
`Optional` means can be `None`, not can be omitted from the arg-list. Mypy will figure out that it can be omitted from the arg-list because there's a default.